.pipeline/templates/api/critterbase.yaml

```diff
@@ -0,0 +1,593 @@
+                        name: ${DATABASE_CREDENTIALS_SECRET}
+                        key: ${DATABASE_USER_KEY}
+                  - name: POSTGRES_DATABASE
+                    valueFrom:
+                      secretKeyRef:
+                        name: ${DATABASE_CREDENTIALS_SECRET}
+                        key: ${DATABASE_NAME_KEY}
+                  - name: PGDATA
+                    value: /var/lib/pgsql/data/pgdata
+                ports:
+                  - containerPort: ${{DATABASE_PORT}}
+                    protocol: TCP
+                imagePullPolicy: Always
+                volumeMounts:
+                  - name: ${APP_NAME}-db
+                    mountPath: /var/lib/pgsql/data
+                terminationMessagePolicy: File
+                image: ${BASE_IMAGE_REGISTRY_URL}/${NAMESPACE_PREFIX}-tools/postgres-postgis:latest
+            restartPolicy: Always
+            terminationGracePeriodSeconds: 30
+            dnsPolicy: ClusterFirst
+            securityContext: {}
+            schedulerName: default-scheduler
+
+    # Database PVC <NAMESPACE_ENVIRONMENT>
+    - kind: PersistentVolumeClaim
+      apiVersion: v1
+      metadata:
+        name: ${APP_NAME}-db
+        namespace: ${NAMESPACE_PREFIX}-${NAMESPACE_ENVIRONMENT}
+        labels:
+          app: ${APP_NAME}
+          name: ${APP_NAME}-db
+          tag: ${TEMPLATE_TAG}
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: ${DB_PVC_SIZE}
+        storageClassName: netapp-block-standard
+        volumeMode: Filesystem
+
+    # DB Build Config
+    - kind: BuildConfig
+      apiVersion: build.openshift.io/v1
+      metadata:
+        name: postgres-postgis
+        namespace: ${NAMESPACE_PREFIX}-tools
+        labels:
+          app: ${APP_NAME}
+          name: postgres-postgis
+          tag: ${TEMPLATE_TAG}
+      spec:
+        nodeSelector: null
+        output:
+          to:
+            kind: ImageStreamTag
+            name: 'postgres-postgis:latest'
+        resources: {}
+        successfulBuildsHistoryLimit: 5
+        failedBuildsHistoryLimit: 5
+        strategy:
+          type: Docker
+          dockerStrategy:
+            env:
+              - name: POSTGRES_PASSSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: ${DEFAULT_POSTGRES_PASSWORD_SECRET}
+                    key: ${DEFAULT_POSTGRES_PASSWORD_KEY}
+            dockerfilePath: .docker/Dockerfile.db
+        postCommit: {}
+        source:
+          type: Git
+          git:
+            uri: ${CRITTERBASE_GITHUB_URL}
+          secrets:
+            - secret:
+                name: etc-pki-entitlement
+              destinationDir: etc-pki-entitlement
+          configMaps:
+            - configMap:
+                name: rhsm-conf
+              destinationDir: rhsm-conf
+            - configMap:
+                name: rhsm-ca
+              destinationDir: rhsm-ca
```

Comment:
You might not need these sections.

These entitlements should get automatically included in newer versions of Openshift, at least if they are using redhat nodes (not entirely sure on the details here). 

Shouldn't hurt to keep them here either, but you could try and see if this still works without them.